tests/blockchain/eth/entities/deployers/test_interdeployer_integration.py

```diff
@@ -97,10 +97,10 @@ def test_deploy_ethereum_contracts(testerchain,
 
     with pytest.raises(BaseContractDeployer.ContractDeploymentError):
         assert policy_manager_deployer.contract_address is constants.CONTRACT_NOT_DEPLOYED
-    assert not policy_manager_deployer.is_deployed
+    assert not policy_manager_deployer.is_deployed()
```

Comment:
Is `PolicyManagerDeployer.is_deployed` no longer a @property method?

Reply:
I added version as parameter to this method